src/views/Home/components/NewsCard/index.jsx

```diff
@@ -27,6 +25,7 @@ const NewsCard = () => {
             </Grid>
           </Grid>
         }
+        className={'gc360_header'}
```

Comment:
```suggestion
        className='gc360_header'
```
Brackets aren't needed, so this should be written like this everywhere you use it